src/components/FormControl/FormControl.stories.ts

```diff
@@ -7,6 +7,9 @@ export default {
     title: 'Forms/Form control',
     component: FormControl,
     argTypes: {
+        autofocus: {
+            control: {type: 'boolean'},
+        },
```

Comment:
Not sure if this is logical to put here, as autofocus only really works if it is set when the component gets loaded, it wont work when setting the value to `true` after the page is already loaded.

What do you think about it?

Reply:
Indeed feels like it will not work.